packages/file-search/src/browser/quick-file-open.ts

```diff
@@ -69,10 +78,12 @@ export class QuickFileOpenService implements QuickOpenModel, QuickOpenHandler {
      */
     protected isOpen: boolean = false;
 
+    protected filterAndRangeQueryDefault = { filter: '', range: { start: { line: 0, character: 0 }, end: { line: 0, character: 0 } } };
+
     /**
-     * The current lookFor string input by the user.
+     * Tracks the user file search filter and location range e.g. fileFilter:line:column or fileFilter:line,column
      */
-    protected currentLookFor: string = '';
+    protected fileSearchQuery: FilterAndRange = this.filterAndRangeQueryDefault;
```

Comment:
We should probably rename the variable to properly represent the type:

```suggestion
    protected filterAndRange: FilterAndRange = this.filterAndRangeQueryDefault;
```

Reply:
Done, This one, it may not be quite clear to a new reader but I it's consistent with the name of the interface so I am OK with it.